doc/releases/migration-guide-4.1.rst

```diff
@@ -238,6 +238,11 @@ GPIO
 * Renamed the device tree property ``off_val`` to ``off-val``.
 * Renamed the device tree property ``on_val`` to ``on-val``.
 * Renamed the ``compatible`` from ``ti,ads114s0x-gpio`` to :dtcompatible:`ti,ads1x4s0x-gpio`.
+* To support the RP2350B, which has many pins, the RaspberryPi-GPIO configuration has
```

Comment:
4.1 feature freeze is past

Reply:
Moved into `doc/releases/migration-guide-4.1.rst`.